pkg/planner/core/casetest/index/index_test.go

```diff
@@ -56,6 +56,57 @@ func TestIndexJoinUniqueCompositeIndex(t *testing.T) {
 	}
 }
 
+func TestIndexMergeFromComposedDNFCondition(t *testing.T) {
```

Comment:
Better to add one more test to check the result is correct not just the plan, for example "TestMVIndexRandom".

Reply:
sure, it will be in my next pull request